src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
+@import './variebles';
+@import './mixins';
+
 body {
   margin: 0;
 }
+
+.card {
```

Comment:
The card styles could be moved to a separate file for better organization. This would make your code more modular and easier to maintain.